### ISSUE TITLE
Problematic behavior by NVDA when changing password for Microsoft account in Windows Eight.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2863]
**Reported by Harich on 2012-12-11 17:31**
If you go to "Change PC Settings" followed by "User" option in Windows Eight, there's an option to "change your password" for Microsoft account (assuming you've configured online log on in Windows Eight). Clicking on to "change your password" presents the option to enter the old (ie. existing) password followed by entering new password. These are of course protected field; however, NVDA speaks the password you're typing. NVDA also loses focus mode.

**Comment by leonardder** [comment]
Anyone on Windows 8 who can try to reproduce this?

**Comment by Adriani90** [comment]
@Harich is this still reproducible?

**Comment by Adriani90** [comment]
Testing on Windows 10 1809 Update with NVDA 2018.4 Beta 2. I cannot reproduce it.
Can someone test this on Windows 8.1? I think there was a fix related to this in NVDA 2018.1 or 2018.2

**Comment by Adriani90** [comment]
Ok we do not have any further updates from users since years, I am closing this one for now. Please comment on it if you are able to reproduce it with NVDA 2019.1.1 on Windows 8.1. Then we can reopen the discussion. Thanks.
